vector/src/main/java/im/vector/app/features/call/service/CallHeadsUpActionReceiver.kt

```diff
@@ -33,9 +34,11 @@ class CallHeadsUpActionReceiver : BroadcastReceiver() {
 
     override fun onReceive(context: Context, intent: Intent?) {
         val webRtcCallManager = context.singletonEntryPoint().webRtcCallManager()
+        val notificationUtils = context.singletonEntryPoint().notificationUtils()
         when (intent?.getIntExtra(EXTRA_CALL_ACTION_KEY, 0)) {
             CALL_ACTION_REJECT -> {
                 val callId = intent.getStringExtra(EXTRA_CALL_ID) ?: return
+                notificationUtils.cancelAllNotifications()
```

Comment:
Should this really cancel all notifications?

Reply:
Yeap, this was for testing purpose. Fixed, here we should cancel certain notification